src/MathOptInterface.jl

```diff
@@ -51,6 +51,13 @@ Read the file `filename` into the instance `instance`. If `m` is non-empty, this
 throw an error.
 
 Supported file types depend on the instance type.
+
+### Note
+
+Once the model is loaded into the model, users can query the variables via
```

Comment:
"Once the model is loaded into the model" is a bit difficult to grasp :-P Maybe "Once `filename`'s model is loaded in to `model`" reads better

Reply:
"Once the contents of the file are loaded into the model" maybe